main/windows/index.ts

```diff
@@ -7,6 +7,8 @@ import { hexToNumber } from 'web3-utils'
 import store from '../store'
 import FrameManager from './frames'
 
+process.env.BUNDLE_LOCATION = process.env.BUNDLE_LOCATION || path.resolve(__dirname, './../../..', 'bundle')
```

Comment:
This line is already in the `main/index` but should be above all the local imports - I think this was broken in the develop merge.

Reply:
Ah yeah, was going to bring that up in the sync -- just needed to get it working for now.